pkg/daemon/on_disk_validation.go

```diff
@@ -156,7 +155,7 @@ func checkV3Files(files []ign3types.File) error {
 			var err error
 			contents, err = dataurl.DecodeString(*f.Contents.Source)
 			if err != nil {
-				return errors.Wrapf(err, "couldn't parse file %q", f.Path)
+				return fmt.Errorf("couldn't parse file %q: %w", f.Path, err)
```

Comment:
Just an FYI: This may cause a merge conflict depending upon whether openshift/machine-config-operator#2874 is merged before this is.